go/cmd/dolt/commands/utils.go

```diff
@@ -303,9 +297,27 @@ func newLateBindingEngine(
 			rawDb.AddEphemeralSuperUser(ed, dbUser, config.ServerHost, "")
 		}
 
+		sqlCtx, err := se.NewDefaultContext(ctx)
+		if err != nil {
+			return nil, nil, nil, err
```

Comment:
does this one need `se.Close()`? should we defer check `if err != nil` for closing every error retur?